pkg/envoy/sds/builder.go

```diff
@@ -20,11 +20,12 @@ type sdsBuilder struct {
 
 	// The DiscoveryRequest contains the certs to generate secrets for
 	// requestedCerts is expected to be a list of either "service-cert:namespace/service" or "root-cert:namespace/service"
-	requestedCerts []string
+	requestedCerts map[string]any
```

Comment:
I haven't seen `any` before; is this a go1.19 alias for `interface{}`?

Reply:
yes, exactly, it's intended to be a more idiomatic name for interface{}